keyboards/donutcables/budget96/readme.md

```diff
@@ -0,0 +1,44 @@
+# budget96
+
+96 key atmega32a keyboard. 
+
+Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)  
+Hardware Supported: Budget96 PCB  
+Hardware Availability: [DonutCables](https://donutcables.com/)  
+
+Make example for this keyboard (after setting up your build environment):
+
+    make donutcables/budget96:default
+
+Flashing
+
+**Reset Key:** Hold down the key located at `K00`, commonly programmed as left control while plugging in the keyboard. You may also hold down the key located at `K50`, commonly programmed as the `Esc` key. 
+
+ps2avr(GB) boards use an atmega32a microcontroller and a different bootloader. It is not flashable using the regular QMK methods. 
+
+Windows: 
+1. Download [HIDBootFlash](http://vusb.wikidot.com/project:hidbootflash).
+2. Place your keyboard into reset. 
+3. Press the `Find Device` button and ensure that your keyboard is found.
+4. Press the `Open .hex File` button and locate the `.hex` file you created.
+5. Press the `Flash Device` button and wait for the process to complete. 
+
+macOS:
+1. Install homebrew by typing the following:   
+    ```
+    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    ```
+2. Install `crosspack-avr`.  
+    ```
+    brew cask install crosspack-avr
+    ```
+3. Install the following packages:
+    ```
+    brew install python
+    pip3 install pyusb
+    brew install --HEAD`https://raw.githubusercontent.com/robertgzr/homebrew-tap/master/bootloadhid.rb
+
+4. Place your keyboard into reset. 
+5. Flash the board by typing `bootloadHID -r` followed by the path to your `.hex` file. 
+
+See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```

Comment:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Reply:
You should change this templates if you have an issue with it =P.

Reply:
I did... back in August. :stuck_out_tongue: 

https://github.com/qmk/qmk_firmware/commit/836efb50c6fa1be9e3189242479e7ae5bc2b3f38#diff-d638158e12c65a8dc7ab6db644a20593